pymms/productevent.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""py-mms product event
+
+ PY-MMS : Product Event Class
+ ============================
+
+ Copyright 2020 MET Norway
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging
+
+from .gomms import GoMMS
+
+logger = logging.getLogger(__name__)
+
+class ProductEvent():
+    """Wrapper class for the go-mms product event.
+    """
+
+    def __init__(self, product="", productSlug="", productionHub=""):
+
+        self._goMMS = GoMMS()
+
+        # Event properties
+        self._eventProduct = product
+        self._eventProductSlug = productSlug
+        self._eventProductionHub = productionHub
+
+        return
+
+    ##
+    #  Getters
+    ##
+
+    @property
+    def product(self):
+        return self._eventProduct
+
+    @property
+    def productSlug(self):
```

Comment:
Needs a rename when hierarchy is fixed.

Reply:
Will update this in a separate PR when the go-mms code is changed.